src/routes/backoffice.ts

```diff
@@ -290,7 +290,7 @@ const generateBookingFromStandingOrder = (standingOrder) => {
     ...standingOrder,
     id: uuid.v4(),
     valuta_date: moment().format("YYYY-MM-DD"),
-    recorded_at: moment().format(),
+    recorded_at: moment().utc().format(),
     booking_date: moment().format("YYYY-MM-DD"),
```

Comment:
why is this the only one in UTC ?

Reply:
288a0533c9d163a85fb38c27a81127eff9ca2f4a